qoqo-qryd/src/api_backend.rs

```diff
@@ -461,6 +464,7 @@ impl APIBackendWrapper {
     /// Args:
     ///     dev (bool): The boolean to set the dev option to.
     ///
+    #[pyo3(text_signature = "($self, dev, /)")]
```

Comment:
Is the `$` a typo? I haven't seen this notation before

Reply:
According to pyo3, it's how a `&self` reference should be handled when defining a pyclass.
https://github.com/PyO3/pyo3/blob/f12f928bd57e61fdf86f7751a3273708c31de31b/guide/src/class.md?plain=1#L860

Reply:
Nice!